tests/test_flake8_trio.py

```diff
@@ -157,70 +157,68 @@ def check_autofix(
     assert added_autofix_diff == autofix_diff_content
 
 
-# TODO: merge with test_eval_anyio with a parametrize
-@pytest.mark.parametrize(("test", "path"), test_files)
-@pytest.mark.parametrize("autofix", [False, True])
-def test_eval(test: str, path: Path, autofix: bool, generate_autofix: bool):
-    content = path.read_text()
-    if "# NOTRIO" in content:
-        pytest.skip("file marked with NOTRIO")
+MAGIC_MARKERS = ("NOANYIO", "NOTRIO", "TRIO_NO_ERROR", "ANYIO_NO_ERROR")
```

Comment:
> somewhat awkward to type hint with repeating the literals twice, feels like it should be possible to get rid of that.

We could define `MAGIC_MARKERS: TypeAlias = Literal[...]` here, and then use `MAGIC_MARKERS.__args__` where you iterate the strings?